src/ec/packages/ec-component-blockquote/blockquote.story.js

```diff
@@ -8,17 +9,49 @@ import data from '@ecl/ec-specs-blockquote/demo/data';
 import blockquote from './ecl-blockquote.html.twig';
 import notes from './README.md';
 
+// Labels for the group ids
+const optionalGroupId = 'Optional elements';
+const requiredGroupId = 'Mandatory elements';
+const useCasesGroup = 'Use cases';
+
+// Preserve the adapted specs.
+const defaultData = { ...data };
+
+// declare your toggled data
+const authorBtnToggler = () => {
+  defaultData.author = defaultData.author ? false : data.author;
+};
+
+const formatQuote = passeddata => {
+  // declare your knobs when you format your data
+  const citation = passeddata.citation
+    ? text('Citation', passeddata.citation, requiredGroupId)
+    : false;
+
+  const author = passeddata.author
+    ? text('Author', passeddata.author, optionalGroupId)
+    : false;
+
+  const newQuote = {
+    author,
+    citation,
+  };
+
+  // Return the full object.
+  return newQuote;
+};
+
 storiesOf('Components/Blockquote', module)
   .addDecorator(withKnobs)
   .addDecorator(withCode)
   .addDecorator(withNotes)
   .add(
     'default',
-    () =>
-      blockquote({
-        author: text('Author name', data.author),
-        citation: text('Citation', data.citation),
-      }),
+    () => {
+      button('show/ hide author', authorBtnToggler, useCasesGroup);
+
+      return blockquote(formatQuote(defaultData));
+    },
     {
       notes: { markdown: notes, json: data },
```

Comment:
You are passing to the note panel the data as it come from the specs (or the adapter), not the one we really pass to the template, therefore it never changes even if you hide the author. (You will remove this button so the problem will not be in place, but in this implementation you can see that the note panel is neither updated when you change the data using the button nor when you use the knobs ;) )